{"name": "test_4[chromium]", "status": "broken", "statusDetails": {"message": "playwright._impl._errors.TimeoutError: Page.goto: Timeout 30000ms exceeded.\nCall log:\nnavigating to \"https://skillbox.ru/\", waiting until \"load\"", "trace": "self = <module_7.TestValidate object at 0x0000025282316C90>\npage = <Page url='https://skillbox.ru/'>\ngo_to_url = <function go_to_url.<locals>.callback at 0x00000252829FC860>\n\n    def test_4(self, page, go_to_url):\n        # Шаг 1: Перейдите на сайт Skillbox\n>       page.goto(\"https://skillbox.ru\")\n\nmodule_7.py:84: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\..\\..\\..\\miniconda3\\Lib\\site-packages\\playwright\\sync_api\\_generated.py:8843: in goto\n    self._sync(\n..\\..\\..\\..\\miniconda3\\Lib\\site-packages\\playwright\\_impl\\_page.py:524: in goto\n    return await self._main_frame.goto(**locals_to_params(locals()))\n..\\..\\..\\..\\miniconda3\\Lib\\site-packages\\playwright\\_impl\\_frame.py:145: in goto\n    await self._channel.send(\"goto\", locals_to_params(locals()))\n..\\..\\..\\..\\miniconda3\\Lib\\site-packages\\playwright\\_impl\\_connection.py:59: in send\n    return await self._connection.wrap_api_call(\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <playwright._impl._connection.Connection object at 0x0000025282103E60>\ncb = <function Channel.send.<locals>.<lambda> at 0x00000252829FC680>\nis_internal = False\n\n    async def wrap_api_call(\n        self, cb: Callable[[], Any], is_internal: bool = False\n    ) -> Any:\n        if self._api_zone.get():\n            return await cb()\n        task = asyncio.current_task(self._loop)\n        st: List[inspect.FrameInfo] = getattr(task, \"__pw_stack__\", inspect.stack())\n        parsed_st = _extract_stack_trace_information_from_stack(st, is_internal)\n        self._api_zone.set(parsed_st)\n        try:\n            return await cb()\n        except Exception as error:\n>           raise rewrite_error(error, f\"{parsed_st['apiName']}: {error}\") from None\nE           playwright._impl._errors.TimeoutError: Page.goto: Timeout 30000ms exceeded.\nE           Call log:\nE           navigating to \"https://skillbox.ru/\", waiting until \"load\"\n\n..\\..\\..\\..\\miniconda3\\Lib\\site-packages\\playwright\\_impl\\_connection.py:514: TimeoutError"}, "parameters": [{"name": "browser_name", "value": "'chromium'"}], "start": 1734622481783, "stop": 1734622511830, "uuid": "2345f20b-7adf-435f-b983-d0badd2273d1", "historyId": "9808811551ec276a6f0c854eb9118b9b", "testCaseId": "2625270973a6db0ba7dc8f0ee0858324", "fullName": "Tests.homework.module_7.TestValidate#test_4", "labels": [{"name": "parentSuite", "value": "Tests.homework"}, {"name": "suite", "value": "module_7"}, {"name": "subSuite", "value": "TestValidate"}, {"name": "host", "value": "Admin"}, {"name": "thread", "value": "10536-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.homework.module_7"}]}